### ISSUE TITLE
高斯-伯努利RBM

### ISSUE THREAD

**Issue by cjlhxls** [fuzimaoxinan/Tensorflow-Deep-Neural-Networks#11]
您好，请问一下在guass-bin的重建过程中，act_function为什么用affine而不是guass?
即 def reconstruction(self,h):
        logits = tf.matmul(h, tf.transpose(self.W)) + self.bv
        prob_v=self.v_func(logits)
        return logits,prob_v

**Comment by fuzimaoxinan** [comment]
因为，公式推导出来就是这样的，p(v|h)服从高斯分布，取这个分布的均值，也就是 linear(h)

发送自 Windows 10 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>应用

________________________________
发件人: layout_toxin <notifications@github.com>
发送时间: Sunday, April 7, 2019 3:48:16 PM
收件人: fuzimaoxinan/Tensorflow-Deep-Neural-Networks
抄送: Subscribed
主题: [fuzimaoxinan/Tensorflow-Deep-Neural-Networks] 高斯-伯努利RBM (#11)


您好，请问一下在guass-bin的重建过程中，act_function为什么用affine而不是guass?
即 def reconstruction(self,h):
logits = tf.matmul(h, tf.transpose(self.W)) + self.bv
prob_v=self.v_func(logits)
return logits,prob_v

―

**Comment by cjlhxls** [comment]
不好意思还有个问题想要问您一下，DBN第一层为高斯-伯努利RBM的话，第二层还是高斯-伯努利RBM吗？或者是第二层应该是伯努利-伯努利型呢？
谢谢！




------------------ 原始邮件 ------------------
发件人: "Fuzz Pan"<notifications@github.com>;
发送时间: 2019年4月7日(星期天) 下午4:12
收件人: "fuzimaoxinan/Tensorflow-Deep-Neural-Networks"<Tensorflow-Deep-Neural-Networks@noreply.github.com>;
抄送: "470564568"<470564568@qq.com>; "Author"<author@noreply.github.com>; 
主题: Re: [fuzimaoxinan/Tensorflow-Deep-Neural-Networks] 高斯-伯努利RBM (#11)



因为，公式推导出来就是这样的，p(v|h)服从高斯分布，取这个分布的均值，也就是 linear(h) 
 
 发送自 Windows 10 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>应用 
 
 ________________________________ 
 发件人: layout_toxin <notifications@github.com> 
 发送时间: Sunday, April 7, 2019 3:48:16 PM 
 收件人: fuzimaoxinan/Tensorflow-Deep-Neural-Networks 
 抄送: Subscribed 
 主题: [fuzimaoxinan/Tensorflow-Deep-Neural-Networks] 高斯-伯努利RBM (#11) 
 
 
 您好，请问一下在guass-bin的重建过程中，act_function为什么用affine而不是guass? 
 即 def reconstruction(self,h): 
 logits = tf.matmul(h, tf.transpose(self.W)) + self.bv 
 prob_v=self.v_func(logits) 
 return logits,prob_v 
 
 ―

**Comment by fuzimaoxinan** [comment]
你想要他是什么那就是什么
